pkg/volume/csi/csi_block.go

```diff
@@ -367,5 +367,17 @@ func (m *csiBlockMapper) TearDownDevice(globalMapPath, devicePath string) error
 		}
 	}
 
+	// remove /data/vol_data.json first, then remove the other dir.
+	dataDir := getVolumeDeviceDataDir(m.specName, m.plugin.host)
+	err = os.RemoveAll(dataDir)
```

Comment:
use Remove() or RemoveAll() here?

Reply:
current dir is like this ~/${pv}/data/vol.json.  Due to vol_data.json left, /data can not be removed use Remove(). Users' data is under ~/${pv}/${link}, so I think RemoveAll() would not harm users' data

Reply:
I've seen CSI drivers that use bind-mounts instead of a symlink. I know, in most cases it will delete just the device file, not content of device itself.

But we've learned the hard way not to use RemoveAll on a directory / file exported to a container (CSI, in this case). I'd prefer to remove the json file + `Remove()` on the dir. If there are any leftovers it should be reported as error of the CSI driver.

Reply:
Thanks, I have change the PR, like this:

1. Remove json file
2. Remove data dir
3. Remove pv dir

PTAL